### ISSUE TITLE
 Completed episode not marked as watched in AniList and Trakt when using "Skip to next" button in Findroid app

### ISSUE THREAD

**Issue by realFPS** [jarnedemeulemeester/findroid#340]
**Describe the bug**
When using the "Skip to next" button in the Findroid app to directly go to the next episode after completing an episode, the completed episode is not marked as watched in AniList and Trakt. However, it is marked as watched in the Jellyfin library. To mark it as watched in AniList and Trakt using findroid, I have to go back after completing the episode, and then it is automatically marked as watched across those services. I have tried this multiple times with the same result. I have also tested it in the web UI and the episodes are marked as watched everywhere.

**To Reproduce**
1. Setup trakt scrobbling and [anilist sync](https://github.com/vosmiic/jellyfin-ani-sync)
2. Watch an episode till maximum resume pecentage setup in jellyfin playback settings using findroid
3. Skip to next episode

**Expected behavior**
Episode should be marked watched across all services

**Screen recordings**
[Using Findroid](https://imgur.com/a/bGYONjq)
[Using Jellyfin web UI](https://imgur.com/a/D32PoAG) (Expected behaviour)

**Device info:**
 - Device: OnePlus 9 pro
 - Android version: 13
 - App version: v0.10.1
 - Jellyfin version: Version: 10.8.9

**Additional context**
I believe this is not a major issue and can be looked into later whenever the developer does not have their hands full. Thank you for your time and efforts.

**Comment by garvongithub** [comment]
I would like to add something, The watch is not registered on those services even after completing the full video and and not pressing the skip button. It is registered on library but not on those services. The only way to register it everywhere is to go back before completing the full video.

**Comment by crabreader** [comment]
The only way I was able to get it to update on AniList from Findroid using the "jellyfin-ani-sync" plugin was by clicking the "Ticking watched updates providers?" checkbox, by doing this it will update when an episode is marked as watched. However it still won't update when watching an episode to the end (even though the episode gets marked as watched).

**Comment by sannidhyaroy** [comment]
> The only way I was able to get it to update on AniList from Findroid using the "jellyfin-ani-sync" plugin was by clicking the "Ticking watched updates providers?" checkbox, by doing this it will update when an episode is marked as watched. However it still won't update when watching an episode to the end (even though the episode gets marked as watched). 

I'm in the same boat. I got it to work with the Web UI only by using the `Ticking watched updates providers`. But this only works with the Web UI. I couldn’t get it to work at all with Findroid. The episode does get marked as watched but it's not synced to AniList. The only way I can sync it to AniList is by making the watched episode as unwatched and then again marking it as watched manually using the ✅ button.


Tldr: Findroid only syncs watched episodes to AniList if the episode is set as watched manually using the ✅ button. Doesn't sync if episode is marked as watched by Jellyfin automatically after watching an episode.

**Comment by sannidhyaroy** [comment]
@jarnedemeulemeester Can you please take a look at this? It's possible that this might have slipped through as it's been a year, since this thread was opened.

**Comment by jarnedemeulemeester** [comment]
A partial fix has been implemented. 
I will close this issue as #869 has more details.
